book-worm-2010/book-worm-fe
# Feature/friends

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds and styles the friendList. It is manually added for now until we can get a backend endpoint to get friends. 

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.
None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/App/App.js` lines: 55-63;
`src/FriendList/FriendList.js ` lines: 1-48;
`src/FriendList/FriendList.scss` lines 1-35;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch' this branch
2. run `npm start`
3. Click on the friends sandhill
4. See your friends

### Steps To Take
>Any questions and possible steps moving forward.
Add functionality and maybe an add friend button
